### DISCUSSION TITLE
Update Libwebp.dll

### DISCUSSION THREAD

**Discussion by Lolo1664** [FreeCAD/FreeCAD#13145] [Discussions that will turn in to issues but need refining]
Hello,

I would like to know if in your raodmap is planned the update of the Libwebp.dll file which goes back to our EDR as presenting a vulnerability

thank you for your interest in my request

Sincerely,

**Comment by luzpaz** [comment]
Can you please open a forum thread in the packaging section of the forum ?

**Comment by ThomasKur** [comment]
We have the same problem and keeping especially Open Source Projects secure is important. I also created an issue #15638 to perhaps host these libraries externally or in separate folders to allow updating them by users if they care about vulnerabilities.
